.github/CODEOWNERS

```diff
@@ -4,7 +4,7 @@
 /avm/ @Azure/avm-core-team-technical-bicep
 /avm/utilities/ @Azure/avm-core-team-technical-bicep
 #/avm/res/aad/domain-service/ @Azure/avm-res-aad-domainservice-module-owners-bicep
-#/avm/res/analysis-services/server/ @Azure/avm-res-analysisservices-server-module-owners-bicep @Azure/avm-core-team-technical-bicep
+/avm/res/analysis-services/server/ @Azure/avm-res-analysisservices-server-module-owners-bicep @Azure/avm-core-team-technical-bicep
```

Comment:
Team must be created

Reply:
And added to the parent team as per https://azure.github.io/Azure-Verified-Modules/specs/shared/#team-memberships

Reply:
@Azure/avm-res-analysisservices-server-module-owners-bicep
@Azure/avm-res-analysisservices-server-module-contributors-bicep

Reply:
@Azure/avm-res-analysisservices-server-module-owners-bicep
@Azure/avm-res-analysisservices-server-module-contributors-bicep -bicep